spring-cloud-azure-samples/spring-cloud-azure-sql-sample/src/main/java/com/example/WebController.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See LICENSE in the project root for
+ * license information.
+ */
+
+package com.example;
+
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+/**
+ * @author Warren Zhu
+ */
+@RestController
+public class WebController {
+
+    private final JdbcTemplate jdbcTemplate;
+
+    public WebController(JdbcTemplate jdbcTemplate) {
+        this.jdbcTemplate = jdbcTemplate;
```

Comment:
Any where in this Project have we auto-configured a bean for `JdbcTemplate` in order to run this sample?

Reply:
No. Auto happening.